File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
       int count=1;
        int i=1;
        for(int j=1;j<nums.size();j++)
        {
            if(nums[j]==nums[j-1])
                continue;
            nums[i]=nums[j];
            i++;
             count++;
        }
        return count;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int j=1,c=1;
        for(int i=1;i<nums.length;i++)
        {
            if(nums[i]==nums[i-1])
            
                continue;
            nums[j]=nums[i];
            j++;
            c++;
        }
        return c;
        
        
    }
}